src/web/data/userMapData.json

```diff
@@ -2011,14 +2061,14 @@
       "University of Nebraska",
       "University of Nebraska at Kearney",
       "University of Nebraska-Lincoln",
-      "University of Nevada",
+      "University of Nevada, Reno",
```

Comment:
y the deletion?

Reply:
had to add Reno to it

Reply:
but University of Nevada has many branches, please check the relevant issue for the previous entry and make sure that the deletion is appropriate.

Reply:
Oh, I just saw it. Will revert this change back as it was